pkg/resource/resource.go

```diff
@@ -93,6 +99,16 @@ func (r *Resource) AppendRefBy(id resid.ResId) {
 	r.refBy = append(r.refBy, id)
 }
 
+// GetRefVar returns vars that refer to current resource
+func (r *Resource) GetRefVar() []string {
```

Comment:
`GetRefVarNames`


it is plural, and doesn't hold the whole Var.

Reply:
thanks, done